core/server/models/post.js

```diff
@@ -560,7 +560,7 @@ Post = ghostBookshelf.Model.extend({
             // these are the only options that can be passed to Bookshelf / Knex.
             validOptions = {
                 findOne: ['columns', 'importing', 'withRelated', 'require'],
-                findPage: ['page', 'limit', 'columns', 'filter', 'order', 'status', 'staticPages', 'absolute_urls'],
```

Comment:
Looked into `staticPages` that seems to be included in query building logic in `processOptions()` method. another suspect was `status` but that is one of the actual column in the db so is needed for querying.

Reply:
Yeah we have to keep `staticPages` ✌🏻

Reply:
Does `absolute_urls` still get passed into the `toJSON` method?

Reply:
@allouis  yes, if you specify them in options it get's passed [here](https://github.com/TryGhost/Ghost/pull/9899/files#diff-bdcddc304f43d5639517a20c8b5d6308R63) and is allowed through [this](https://github.com/TryGhost/Ghost/pull/9899/files#diff-bdcddc304f43d5639517a20c8b5d6308R42)

Reply:
:+1: I probably should have known that :sweat_smile: :